ops/pebble.py

```diff
@@ -1926,3 +1926,30 @@ def _websocket_url(self, task_id: str, websocket_id: str) -> str:
         base_url = self.base_url.replace('http://', 'ws://')
         url = '{}/v1/tasks/{}/websocket/{}'.format(base_url, task_id, websocket_id)
         return url
+
+    def send_signal(self, sig: typing.Union[int, str], services: typing.List[str]):
+        """Send the given signal to the list of services named.
+
+        Args:
+            sig: Name or number of signal to send, e.g., "SIGHUP", 1, or
+                signal.SIGHUP.
```

Comment:
is signal.SIGHUP a string or int? Otherwise should it be in the union typing?

Reply:
Yeah, it's a bit weird. `signal.SIGxyz` are defined as an `IntEnum` which basically means the values are an `int` subclass, kind of like `type Signal int` in Go.

```python
>>> isinstance(signal.SIGHUP, int)
True
```